src/app/tab2/tab2.page.html

```diff
@@ -1,8 +1,6 @@
 <ion-header [translucent]="true">
   <ion-toolbar>
-    <ion-title>
-      Photo Gallery
-    </ion-title>
+    <ion-title> Photo Gallery </ion-title>
```

Comment:
Is it a best practice to write titles between blocks, or is there more too it than just that?

Reply:
In general makes it easier to read and edit, I use a beautifier extension on vs, which basically orders the code automatically